app/application.py

```diff
@@ -1069,6 +1094,61 @@ def clean_up(tmp_dir):
             raise
 
 
+def save_local_json(json_data, filename):
```

Comment:
why is this only called for deepgram?

Reply:
Because for whisper, a separate function takes care of storing the `.srt` file.